docs/Glossary.md

```diff
@@ -1,3 +1,46 @@
 # Glossary
 
-See [Terminologies](/introduction/Terminologies.html).
+## App
```

Comment:
I think it's easier to find reference if Glossary of terms is on the root, rather than having to go to `docs` -> `terminologies`?